packages/react-components/react-conformance-griffel/package.json

```diff
@@ -45,6 +45,7 @@
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
+      "node": "./lib-commonjs/index.js",
```

Comment:
Should there be an ESM export for this package?

Reply:
The migration generator adds an ESM export based on if the `package.json` contains a `module` field. `react-conformance-griffel` does not have one and I'm not particularly sure if that was intentional or not. If it wasn't intentional, we can certainly add it in

Reply:
Definitely not a blocker, just stood out to me. FYI, @layershifter since you likely can give us a definitive answer.